src/twig.expression.js

```diff
@@ -576,14 +576,14 @@ module.exports = function (Twig) {
         },
         {
             type: Twig.expression.type.slice,
-            regex: /^\[(\d*:\d*)\]/,
+            regex: /^\[(-?\w*:-?\w*)\]/,
```

Comment:
It looks like you're adding support for negative values here, but there are no tests for this. Please add tests.